kie-remote/src/main/java/org/kie/remote/command/EventInsertCommand.java

```diff
@@ -18,27 +18,30 @@
 
 import java.io.Serializable;
 
-public class EventInsertCommand extends WorkingMemoryActionCommand implements VisitableCommand,
-                                                                         Serializable {
+public class EventInsertCommand extends WorkingMemoryActionCommand implements VisitableCommand, Serializable {
 
-    private Object object;
+    private Serializable object;
 
-    public EventInsertCommand(){}
+    public EventInsertCommand() { /*For serialization*/}
 
-    public EventInsertCommand( Object object, String entryPoint ) {
+    public EventInsertCommand(Object object, String entryPoint) {
```

Comment:
I think it is better tha change object type to `Serializable` instead of just do an internal cast

Reply:
ok, going to change the stack of calls